src/server/system_services/bucket_server.js

```diff
@@ -1932,10 +1932,16 @@ async function get_bucket_replication(req) {
     if (!replication_id) throw new RpcError('NO_REPLICATION_ON_BUCKET');
 
     const replication = await replication_store.instance().get_replication_by_id(replication_id);
-    const bucket_names_replication = _.map(replication, rule =>
+    const bucket_names_replication = _.map(replication.rules, rule =>
         ({ ...rule, destination_bucket: system_store.data.get_by_id(rule.destination_bucket).name }));
 
-    return bucket_names_replication;
+    let { logs_location } = replication.aws_log_replication_info ? replication.aws_log_replication_info : undefined;
+    const res = {
+        rules: bucket_names_replication,
+        log_replication_info: logs_location
+    };
```

Comment:
if `aws_log_replication_info` is undefined, I think your code will fail.
you can use optional chaining, which is supported in node 14, or you can just use a simple `if`

with optional chaining
```suggestion
    const res = {
        rules: bucket_names_replication
        log_replication_info: replication?.aws_log_replication_info?.logs_location
    }
```


or:
```suggestion
    const res = {
        rules: bucket_names_replication
    }
    if (replication.aws_log_replication_info) {
        res.log_replication_info = aws_log_replication_info.logs_location
    }
```